apps/epic-web/src/inngest/functions/stripe/checkout-completed.ts

```diff
@@ -0,0 +1,18 @@
+import {inngest} from 'inngest/inngest.server'
+import {STRIPE_CHECKOUT_COMPLETED_EVENT} from '@skillrecordings/inngest'
+
+export const stripeCheckoutCompleted = inngest.createFunction(
+  {id: `stripe-checkout-completed`, name: 'Stripe Checkout Completed'},
+  {event: STRIPE_CHECKOUT_COMPLETED_EVENT},
+  async ({event, step}) => {
+    const {quantity} = event.data
+
+    const teamPurchase = quantity > 1
```

Comment:
I'll probably add an entirely different function or rename this one to handle the team purchaser flow